doc-content/enterprise-only/processes/viewing-process-instances-variables-basic-filters-in-bc-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='viewing-process-instances-variables-basic-filters-in-bc-proc']
+= Viewing process instance variables using basic filters
+
+{CENTRAL} provides basic filters to view process instance variables. You can view the process instance variables of the process as columns using *Show/hide columns*.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the filter icon on the left of the page to expand the *Filters* panel.
+. In the *Filters* panel, select the *Definition Id*.
++
+The filter is applied on the current process instance list.
+. Click *Show/hide columns* on the upper right of the process instances list and the process instance variables of the specified process id will be displayed.
```

Comment:
s/Click *Show/hide columns* on the upper right of the process instances list and the process instance variables of the specified process id will be displayed./Click *Show/hide columns* on the upper right of the process instances list. The process instance variables of the specified process id is displayed.

Reply:
Done.